safe.py

```diff
@@ -735,6 +735,14 @@ def __repr__(self):
   def copy(self):
     # Create a new instance
     copy_inst = SafeDict(self.__under__)
+    
+    # Fix for recursion depth reached issue #97 when copying and printing a   
```

Comment:
In the past I've come across issue numbers that refer to Seattle's old issue tracker.  Newcomers might come across these issue numbers and be confused about where to find them.  Consider linking to the actual issue on GitHub.